backend/spellbook/management/s3_upload.py

```diff
@@ -3,30 +3,26 @@
 import gzip
 import logging
 
-ACCESS_KEY = os.environ.get('AWS_ACCESS_KEY_ID', None)
-SECRET_KEY = os.environ.get('AWS_SECRET_ACCESS_KEY', None)
 BUCKET = os.environ.get('AWS_S3_BUCKET', None)
 
 
 def can_upload_to_s3() -> bool:
     try:
         import boto3
         try:
-            boto3.client('s3', aws_access_key_id=ACCESS_KEY,
-                        aws_secret_access_key=SECRET_KEY)
+            boto3.client('s3')
```

Comment:
I'm not confident this does (or did) anything. Only when making a method call does it raise `NoCredentialsError`